pkg/apis/security/v1alpha1/types.go

```diff
@@ -152,6 +160,12 @@ type NetworkPolicyPeer struct {
 	Group string `json:"group,omitempty"`
 }
 
+
+type Namespaces struct {
```

Comment:
We should not just name it as Namespaces. NetworkPolicyPeerNamespaces (I know it is long)?

Reply:
Does `PeerNamespaces` work?

Reply:
PeerNamespaces sounds good to me!